### DISCUSSION TITLE
[Unknown Mod][Captured Artifact window] Addition of the name and graphic of the spell contained in the spell scroll

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#7039] [Ideas]
The name of the spell contained in the spell scroll will be indicated below the spell scroll sprite:

"Spell Scroll: Blind"

Instead of just:

"Spell Scroll"

The graphic of the spell contained in the spell scroll will be displayed on the right next to the spell scroll sprite (the group of the two graphics centered horizontally in the window).
Right-clicking and holding on the spell graphic displays the spell information window.
Right-clicking and holding on the spell scroll graphic displays the spell scroll information window.
In the Captured Artifact window, a left click on the artifact graphic (and spell graphic) displays the artifact information window with an "OKAY" button (same operation as in the Magic Book for spells).
The image is for explanation only.

In fheroes2:

![Capture window spell scroll et Blind centrés 01](https://user-images.githubusercontent.com/43583994/233140677-9699dae9-7f1d-4be8-aed9-cf39897d67ee.png)

This is not optional.
This can also be done by default in fheroes2.

**Comment by Branikolog** [comment]
Hi, @LeHerosInconnu 
An addition to show spell name near "Spell Scroll" is a nice idea. I personally find the proposed visual implementation as debatable (there's too many icons at one time). Showing spell name at least in this window would be definitely a step to comfortable UI. Will reverse order be grammatically correct? I find a form without colon more pleasing to eye: "Blind Spell Scroll".

**Reply by LeHerosInconnu** [reply]
Hello @Branikolog,

> Hi, @LeHerosInconnu
> An addition to show spell name near "Spell Scroll" is a nice idea. I personally find the proposed visual implementation as debatable (there's too many icons at one time).

I had also thought of only displaying the spell graphic, but the Spell Scroll graphic is so nice. :)
Maybe a sort of mix of both, I don't know at this time.

> Showing spell name at least in this window would be definitely a step to comfortable UI.
> Will reverse order be grammatically correct? I find a form without colon more pleasing to eye: "Blind Spell Scroll".

I really don't know.
I find it acceptable in a sentence, but in this indicative case it would seem strange to me.
I had also thought about:
Spell Scroll "Blind"
But I preferred the colon in the end, because it is used a lot in the interface. :)

**Comment by LeHerosInconnu** [comment]
I think it's okay to have only the Spell Scroll sprite.

For the text, why not:

"Spell Scroll contains Blind"

Or:

"Spell Scroll with Blind"

**Comment by LeHerosInconnu** [comment]
Or simply the name of the spell:

"Blind"

Should also be added to these windows:

In fheroes2:

![Spell scroll name 001](https://github.com/ihhub/fheroes2/assets/43583994/0642f02f-ad0b-4677-abd3-3f31f70d483b)

![Spell scroll name 002](https://github.com/ihhub/fheroes2/assets/43583994/0a5ad362-6bd2-44d2-b216-0c548271a2f2)

**Comment by LeHerosInconnu** [comment]
These windows are also concerned:

In fheroes2:

![Hero screen spell scroll window 01](https://github.com/ihhub/fheroes2/assets/43583994/03af4ada-2302-4eb8-81d1-7ee09a675518)

![Hero screen spell scroll window 02](https://github.com/ihhub/fheroes2/assets/43583994/1cefa031-4199-484a-9a7f-bc54d26d30a0)
